frontend/src/components/EnSavoirPlus.jsx

```diff
@@ -1,5 +1,17 @@
 import React from "react";
+import "./styles/EnSavoirPlus.scss";
```

Comment:
Please rename your scss file to LearnMore.scss

Reply:
Checked